packages/zowe-explorer/package.json

```diff
@@ -1803,6 +1833,11 @@
           "type": "boolean",
           "description": "%zowe.cliLoggerSetting.presented%",
           "scope": "application"
+        },
+        "zowe.jobs.pollInterval": {
+          "type": "integer",
+          "description": "%zowe.pollInterval.info%",
+          "scope": "window"
```

Comment:
I will definitely update the default value to something valid, I forgot to update that after making some changes to how the setting works.

As for the 1000ms part, should we make that the minimum value? So that nothing under 1000ms will be accepted?

Reply:
I've updated the UX to present an input box for the polling interval, rather than a quick pick - that way users can press Enter to confirm the interval, or they can update it without changing the dialog.

I've also added a minimum polling interval of 1000ms - willing to adjust if desired 🙂